js/therock.js

```diff
@@ -925,6 +934,46 @@ module.exports = class therock extends Exchange {
         return this.parseTransactions (depositsAndWithdrawals, currency, since, limit);
     }
 
+    async withdraw (code, amount, address, tag = undefined, params = {}) {
+        [ tag, params ] = this.handleWithdrawTagAndParams (tag, params);
+        await this.loadMarkets ();
+        const currency = this.currency (code);
+        const request = {
+            'currency': currency['id'],
+            'destination_address': address,
+            'amount': amount,
```

Comment:
Let's use `currencyToPrecision` to format it properly. It returns a string. We can wrap it in parseFloat if they expect a float value.